modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/DataEngineLayoutBuilderHandler.es.js

```diff
@@ -15,6 +15,25 @@
 export default function ({defaultLanguageId, namespace}) {
 	const form = document.getElementById(`${namespace}fm`);
 
+	const clearNameInputIfNeeded = () => {
+		const inputComponent = Liferay.component(`${namespace}name`);
```

Comment:
`Liferay.component` is an API that can return `null`, so you would like to either defend with `if (inputComponent)` or use the `Liferay.componentReady(...).then(inputComponent => { // use inputComponent here })` promise-based API

Reply:
Thanks for the feedback @jbalsas!, I have already added it to the code to prevent possible errors.